Libraries/LibWeb/HTML/HTMLButtonElement.cpp

```diff
@@ -29,6 +31,22 @@ void HTMLButtonElement::initialize(JS::Realm& realm)
     Base::initialize(realm);
 }
 
+void HTMLButtonElement::adjust_computed_style(CSS::ComputedProperties& style)
+{
+    // https://html.spec.whatwg.org/multipage/rendering.html#button-layout
+    // If the computed value of 'display' is 'inline-grid', 'grid', 'inline-flex', 'flex', 'none', or 'contents', then behave as the computed value.
+    auto display = style.display();
+    if (display.is_flex_inside() || display.is_grid_inside() || display.is_none() || display.is_contents()) {
+
```

Comment:
Nit: Put a `// No-op` comment or something in here, so it doesn't look like the code is just missing.